### DISCUSSION TITLE
Pretrained weights and pixel-F1 Score not available currently.

### DISCUSSION THREAD

**Discussion by Better-Than-Never** [open-edge-platform/anomalib#552] [Feature Requests]
First,Thank you guys for this great project!

I'm doing a benchmark related to pixel-level F1 score accross the models, sadly my hardware situation is a little bit on shortage since I'm still in collage.
And I wonder if you could provide the pretrained model of the methods in your projects.
Alternatively, if the pretrained model could not be provided for some reason, is it possible for you guys to provide the full benchmark results regarding each models. 
Currently only the image F1-score is provided, while the pixel-level F1-score is not, and I'm hoping you could release the full results.
Thanks again for the project!

**Comment by samet-akcay** [comment]
Hi @Better-Than-Never, apologies for our late reply. We'll investigate if we could share the weights. Given that MVTec has 15 categories, number of models that we need to share would be quite huge. 

Meanwhile, we will be running the benchmark results and update the readme.
